spec/calendar.html

```diff
@@ -716,13 +721,17 @@ <h1>CalendarEquals ( _one_, _two_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-consolidatecalendars" aoid="ConsolidateCalendars">
-      <h1>ConsolidateCalendars ( _one_, _two_ )</h1>
-      <p>
-        The abstract operation ConsolidateCalendars takes two arguments _one_ and _two_, which must be Objects.
-        It returns the calendar that should take priority when combining two Temporal objects with different calendars, or throws an exception if the calendars cannot be combined.
-        It performs the following steps:
-      </p>
+    <emu-clause id="sec-temporal-consolidatecalendars" type="abstract operation">
+      <h1>
+        ConsolidateCalendars (
+          _one_: a String or Object,
+          _two_: a String or Object,
+        ): either a normal completion containing either a String or Object, or a throw completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It returns the calendar that should take priority when combining two Temporal objects with different calendars, or throws an exception if the calendars cannot be combined.</dd>
+      </dl>
       <emu-alg>
         1. If _one_ and _two_ are the same Object value, return _two_.
         1. Let _calendarOne_ be ? ToTemporalCalendarIdentifier(_one_).
```

Comment:
Passing thought: I wonder if redundancy with CalendarEquals should be reduced? Probably not, since I think it would require some contortions like parameterizing "iso8601" here (e.g., `If Completion(ConsolidateCalendars(_one_, _two_)).[[Type]] is ~normal~, return *true*. Otherwise, return *false*.` in CalendarEquals vs. `Let _calendar_ be ? ConsolidateCalendars(_dateTime_.[[Calendar]], _plainDate_.[[Calendar]], *"iso8601"*).` in Temporal.PlainDateTime.prototype.withPlainDate).

Reply:
I'm skeptical about that. For one thing, checking the completion type would mean that CalendarEquals would return false if ToTemporalCalendarIdentifier throws, instead of throwing as it currently does.